### DISCUSSION TITLE
Error while deploying a contract

### DISCUSSION THREAD

**Discussion by Tanvi1700** [Cyfrin/foundry-full-course-cu#2508] [Q&A]
when I deploy the contract, I'm getting this error I've tried to increase the gas price but still I'm getting this error.
and what's the right way of increasing gas price?

 
creation of SimpleScript pending...
creation of SimpleScript errored: Error occurred: Failed to serialize transaction: toAddressIsNull.

Failed to serialize transaction: toAddressIsNull

You may want to cautiously increase the gas limit if the transaction went out of gas.

**Comment by EngrPips** [comment]
Hello @Tanvi1700, Please what command are you running to deploy your contract, and what chain are you running on?

**Reply by Tanvi1700** [reply]
when I click on deploy button a notification pops up specifying Gas estimation failed and I'm following every instruction given in the video tutorial 

![deploy](https://github.com/user-attachments/assets/980eb23c-1475-4b52-b809-1629a9379225)
![gaserror](https://github.com/user-attachments/assets/1d54dd3e-4721-4ecb-a388-2705e8910461)

**Reply by EngrPips** [reply]
Can you copy and send your `SimpleScript` file to me? Let me try to deploy it on my end.

**Reply by Tanvi1700** [reply]
`



//SPDX-License-Identifier: MIT
pragma solidity 0.8.26;

contract SimpleScript {

    uint256 myFavoriteNumber;

    struct Person {
        uint256 favoriteNumber;
        string name;
    }

    Person[] public listOfPeople;

    mapping (string => uint256) public nameToFavoriteNumber;

    function store(uint256 _favoriteNumber) public {
        myFavoriteNumber = _favoriteNumber;
    }

    function retrieve() public view returns(uint256){
        return myFavoriteNumber;
    }

    function addPerson(string memory _name, uint256 _favoriteNumber) public {
        listOfPeople.push( Person(_favoriteNumber, _name) );
        nameToFavoriteNumber[_name] = _favoriteNumber;
    }
    
}




`

**Comment by EngrPips** [comment]
I successfully deployed your `SimpleScript` contract on Foundry, without any problem

<img width="1440" alt="Screenshot 2024-09-20 at 16 44 39" src="https://github.com/user-attachments/assets/7868b133-f960-4f1e-b4f0-24f05240d742">
